src/index.tsx

```diff
@@ -233,18 +233,6 @@ function renderScene(
     }
   });
 
-  let minX = Infinity;
-  let maxX = 0;
-  let minY = Infinity;
-  let maxY = 0;
-
-  elements.forEach(element => {
-    minX = Math.min(minX, getElementAbsoluteX1(element));
-    maxX = Math.max(maxX, getElementAbsoluteX2(element));
-    minY = Math.min(minY, getElementAbsoluteY1(element));
-    maxY = Math.max(maxY, getElementAbsoluteY2(element));
-  });
-
```

Comment:
Why did you delete all of this, is it unused?

Reply:
Yes, there are no `minX`, `maxX`, `minY`, `maxY` usages elsewhere